crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -1926,10 +1926,10 @@ pub fn integer_squareroot(n: u64) -> u64 {
     }
 
     let mut x = n;
-    let mut y = (x + 1) / 2;
+    let mut y = x.div_ceil(2);
     while y < x {
         x = y;
-        y = (x + n / x) / 2;
+        y = (x + n / x).div_ceil(2);
```

Comment:
This changes its behavior. e.g., 7 / 2 == 3 while 7.div_ceil(2) == 4

Reply:
I tested locally and Clippy only recommends we switch `(x + 1) / 2` to `x.div_ceil(2)`. It doesn't recommend we switch this to use `div_ceil(2)` good catch jihoon